canonical-web-and-design/ubuntu.com
# Add address to 12.10 section of ubuntu-advantage-service-terms

## Done
- Add address to 12.10 section of ubuntu-advantage-service-terms
- *Drive by*: Pass the local copydoc value to the parent template correctly

## QA
- Go to /legal/ubuntu-advantage-service-terms
- Check 12.10 copy matches the [copy doc](https://docs.google.com/document/d/1-sraH8XHez4uDQy8FdK3bzTn_PIduzWJ-23lpnNrzMc/edit)
- Use the [copy docs browser plugin](https://github.com/canonical-web-and-design/ubuntu-copy-docs) or view-source and check the meta copydoc value is "https://docs.google.com/document/d/1-sraH8XHez4uDQy8FdK3bzTn_PIduzWJ-23lpnNrzMc/edit"
